### PR TITLE
feat: add ai/ml model and embedding components

### PR DESCRIPTION
**Pull request by jordanrfrazier**
Adds components for the AI/ML Chat Completion API and Embeddings API. 

Note that while different models support different parameters, currently the AI/ML framework supports this static list, which is what is provided. In the future, we'll ideally have a dynamic set of inputs depending on which model is selected. 

TODO
* Icon

### CODE REVIEW

File: src/backend/base/langflow/components/embeddings/AIMLEmbeddings.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from langflow.components.embeddings.util.AIMLEmbeddingsImpl import AIMLEmbeddingsImpl
+from langflow.field_typing import Embeddings
+from langflow.inputs.inputs import DropdownInput
+from langflow.io import SecretStrInput
+
+
+class AIMLEmbeddingsComponent(LCEmbeddingsModel):
+    display_name = "AI/ML Embeddings"
+    description = "Generate embeddings using the AI/ML API."
+    icon = "OpenAI"  # TODO: icon
```
Comment (jordanrfrazier) [inline src/backend/base/langflow/components/embeddings/AIMLEmbeddings.py:13]: todo

File: src/backend/base/langflow/components/models/AIMLModel.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    DropdownInput,
+    StrInput,
+)
+import httpx
+
+
+class AIMLModelComponent(Component):
+    display_name = "AI/ML API"
+    description = "Generates text using the AI/ML API"
+    icon = "ChatInput"  # TODO: Get their icon.
```
Comment (jordanrfrazier) [inline src/backend/base/langflow/components/models/AIMLModel.py:22]: todo

**Review by nicoloboschi — APPROVED** [review]
LGTM

File: src/backend/base/langflow/components/models/AIMLModel.py (hunk 2 above)
Comment (anovazzi1) [reply src/backend/base/langflow/components/models/AIMLModel.py:21]: done

File: src/backend/base/langflow/components/embeddings/AIMLEmbeddings.py (hunk 1 above)
Comment (anovazzi1) [reply src/backend/base/langflow/components/embeddings/AIMLEmbeddings.py:12]: done

Comment (anovazzi1) [reply src/backend/base/langflow/components/embeddings/AIMLEmbeddings.py]: dpne

**Pull request merged**